includes/class-dispatcher.php

```diff
@@ -73,7 +86,86 @@ public static function process_outbox( $id ) {
 			$activity->set_object( $activity->get_object()->get_id() );
 		}
 
-		self::send_activity_to_followers( $activity, $actor_id, $outbox_item );
+		if ( ! self::should_send_to_followers( $activity, $actor_id, $outbox_item ) ) {
+			return;
+		}
+
+		// Send to mentioned and replied-to users. Everyone other than followers.
+		self::send_to_interactees( $activity, $actor_id, $outbox_item );
```

Comment:
`send_to_interactees` should be called even if `should_send_to_followers` is false, because even private messages should be sent to the mentioned users.

Reply:
Nice catch! Fixed in [9fe47dd](https://github.com/Automattic/wordpress-activitypub/pull/1262/commits/9fe47dd29a93f895fe380a7be05ec60f3c4351e9).